dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -4,8 +4,10 @@
 
 FROM {{ .from }}
 
+# Installing jq needs to be on a separate layer to be installed correctly
```

Comment:
This is all in the same layer as its a single `RUN` operation in the Dockerfile.

I would update the comment to:

```suggestion
# Installing jq needs to be installed after epel-release and cannot be in the same yum install command.
```